.env.default

```diff
@@ -46,6 +46,7 @@ SEND_EMAILS_JOB_CRON="0 * * * *"
 NEW_RELIC_LICENSE_KEY=
 NEW_RELIC_APP_NAME=prod.yourapp.yourdomain.com
 NEW_RELIC_ENABLED=false
+#NEW_RELIC_LABELS=
```

Comment:
This has a `#` at the start in case the user doesn't want to use it. It gets magically injected into new relic (not my choice it's theirs lib)

Reply:
In this case I would suggest to just remove it from the default env, I mean, it is already described on the docs

Reply:
yeah - i agree Carlos.

Removed it.